Source/Krypton Components/Krypton.Toolkit/Palette Builtin/Visual Studio/Official Themes/2012/PaletteVisualStudio2012DarkMode.cs

```diff
@@ -0,0 +1,15 @@
+﻿using System;
```

Comment:
New files need license and the removal of the `using`'s to follow the same pattern as the other files